etest/tests.py

```diff
@@ -106,7 +105,7 @@ def environment(self):
 
     def run(self):
         """Run the test."""
-        docker.pull(self.base_docker_image)
+        # docker.pull(self.base_docker_image)
```

Comment:
I'd say comments indicate an issue and we should just remove this line.

Reply:
This is for development only. We should remove it before merging.

Reply:
Why is this required for development?  You should be able to pull that image once before you start developing and get the same result, no?

Reply:
If you leave it in, its gonna try to pull from dockerhub, which is terribly outdated.

Reply:
Ah, perhaps we need a way to indicate that docker should pull this image or not?